RefactoringPracticeI/GameInventory/Inventory/IHandlerItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace RefactoringExerciseI.Inventory
+{
+    public interface IHandlerItem
+    {
+        /*
+        * Updates the quality of an item. 
```

Comment:
Subjective: this might feel like overdocument the method whose name is self explanatory, unless it has complex operations within (for instance this documentation mentions quality only, but we know we affect Sell In property too)